src/client/features/search/landing-box.js

```diff
@@ -69,12 +71,12 @@ const getNcbiInfo = (ids,genes) => {
     const geneResults=result.result;
     return geneResults.uids.map(gene=>{
       const originalSearch = ids[gene];
-      const links=idToLinkConverter(genes[originalSearch]);
+      const links= genes ? idToLinkConverter(genes[originalSearch]) : [];
```

Comment:
You have the HGNC symbol - is it possible to wrap this data for idToLinkConverter?